projects/helper/env.js

```diff
@@ -7,7 +7,7 @@ const DEFAULTS = {
   EVMOS_MULTICALL_CHUNK_SIZE: "3", // evmos reduced gas limit, this is a workaround to make multicall work
   STARKNET_RPC: 'https://starknet-mainnet.public.blastapi.io',
   COVALENT_KEY: 'ckey_72cd3b74b4a048c9bc671f7c5a6',
-  SOLANA_RPC: 'https://mainnet.helius-rpc.com/?api-key=0109717a-77b4-498a-bc3c-a0b31aa1b3bf',
+  SOLANA_RPC: 'https://mainnet.helius-rpc.com/?api-key=7873b843-941b-45f1-a402-6a41f370ea30',
```

Comment:
can you revert this? we are fine/use our own rpc but this leaks your api key to public

Reply:
@g1nt0ki and it doesn't work rn